### ISSUE TITLE
Didn't actually lose tracking

### ISSUE THREAD

**Issue by philerooski** [Sage-Bionetworks/neurolincsdreamchallenge#9]
These cells have a TimePoint marked as Lost_Tracking = TRUE, but still have values recorded for the next timepoint.

```
   Experiment   ObjectTrackID Well 
   <chr>                <int> <chr>
 1 LINCS062016B             6 A11  
 2 LINCS062016B            57 B12  
 3 LINCS062016B             6 B4   
 4 LINCS062016B            18 B4   
 5 LINCS062016B            33 B6   
 6 LINCS062016B            18 C3   
 7 LINCS062016B             8 C4   
 8 LINCS062016B            18 D10  
 9 LINCS062016B            45 D4   
10 LINCS062016B            34 G11  
11 LINCS062016B             8 G8   
12 LINCS062016B            57 H8 
```

For example:

```
     Experiment Well ObjectTrackID TimePoint Live_Cells Lost_Tracking
1  LINCS062016B  A11             6         1       TRUE         FALSE
2  LINCS062016B  A11             6         2       TRUE         FALSE
3  LINCS062016B  A11             6         3       TRUE         FALSE
4  LINCS062016B  A11             6         4       TRUE         FALSE
5  LINCS062016B  A11             6         5       TRUE         FALSE
6  LINCS062016B  A11             6         6       TRUE         FALSE
7  LINCS062016B  A11             6         7       TRUE         FALSE
8  LINCS062016B  A11             6         8       TRUE         FALSE
9  LINCS062016B  A11             6         9       TRUE         FALSE
10 LINCS062016B  A11             6        10       TRUE         FALSE
11 LINCS062016B  A11             6        11       TRUE          TRUE
12 LINCS062016B  A11             6        12       TRUE          TRUE
13 LINCS062016B  A11             6        13       TRUE          TRUE
14 LINCS062016B  A11             6        14       TRUE          TRUE
```

**Comment by jaslinkalra** [comment]
So a lot of this is happening because the cell masks get renumbered at the later timepoint for the ones numbered at T0. This image segmentation was done in Pipeline Pilot and instead of designating a higher number like 250, 300, 500, pipeline pilot resets and starts from 1 again at a certain threshold of numbers that it can assign on cell masks. I'm currently re-curating again and should be done by the end of today

**Comment by jaslinkalra** [comment]
Correct curation here- https://www.synapse.org/#!Synapse:syn18174013

**Comment by kdaily** [comment]
@phil did other things addressed (or our decision to replace entire tracks instead of specific rows) fix this issue?

Is this related to #6?

**Comment by philerooski** [comment]
This is probably a separate problem from #6 since (like the example track above) there might not be a gap. If Lost_Tracking is TRUE it's telling us "the next TimePoint is missing", but then we see that there is still a row for the next TimePoint.

I replaced the tracks in the original file with the tracks provided by Jaslin. I didn't check if this resulted in missing rows after the initial Lost_Tracking = TRUE, but I did implement the rule 
> If Lost_Tracking is TRUE, all subsequent Lost_Tracking values must be TRUE.

and
> If Lost_Tracking is TRUE, Live_Cells must be NA.

which would have overwritten the values of the subsequent rows anyways.
